app/src/Event/EventController.php

```diff
@@ -291,6 +302,85 @@ public function submit()
     }
 
     /**
+     * Action used to display a form to edit an event and with which the form can be submitted
+     *
+     * @return void
+     */
+    public function edit($friendly_name)
+    {
+        $request = $this->application->request();
+
+        $eventApi = $this->getEventApi();
+        $event    = $eventApi->getByFriendlyUrl($friendly_name);
+        if (! $event) {
+            $this->redirectToListPage();
+        }
+
+        if (! $this->userIsAllowedToEdit($event)) {
+            $this->redirectToDetailPage($event->getUrlFriendlyName());
+        }
+
+        /** @var FormFactoryInterface $factory */
+        $factory = $this->application->formFactory;
+        $form    = $factory->create(new EventFormType(), $event->setTags(implode(', ', $event->getTags())));
```

Comment:
I'm not happy about rewriting the way tags are stored internally from an array to a csv.
I think that the correct Symfony2 Forms solution to this problem is to implement a "DataTransformer" like we did for dates here: https://github.com/joindin/joindin-web2/blob/master/app/src/Form/DataTransformer/DateTransformer.php